README.md

```diff
@@ -1,9 +1,54 @@
-# pm-idm
+# IDM - Identity Manager
 
-Project Management for the IDM Project
+The Identity Manager - IDM for short - is a unified digital wallet based on open-standards that aims to support multiple decentralized identities.
 
-[![#ipfs-identity](https://img.shields.io/badge/irc-%23ipfs--identity-brightgreen.svg)](https://webchat.freenode.net/?channels=ipfs-identity)
+## Documents
 
-## Current status
+Here is a list of some useful documents. If you're interested in collaborating, please check [Contributing](#contributing).
 
-[![Waffle.io - Columns and their card count](https://badge.waffle.io/ipfs-shipyard/pm-idm.svg?columns=all)](https://waffle.io/ipfs-shipyard/pm-idm)
+[💡 Concept](docs/idm-concept.md)
+[🛠 IDM Breakdown](https://docs.google.com/document/d/1g0TjSPjEM4pryPwJTGhIeE4DBsj-VJpz_JqbfllJUgA)
+[🗓 IDM Workplan](https://docs.google.com/spreadsheets/d/1Venqgkcao2Lcje0mkCxr9u0H037aC3H5IxMfsoaeMoE)
+
+## Team
+- [André Cruz](https://github.com/satazor) - Software engineer
+- [André Sousa](https://github.com/andreforsousa) - Designer
+- [Marco Oliveira](https://github.com/marcooliveira) - Product engineer
+- [Paulo Marcos](https://github.com/paulobmarcos) - Software engineer
+- [Pedro Santos](https://github.com/PedroMiguelSS) - Software engineer
+- [Pedro Teixeira](https://github.com/pgte) - Adviser
+
+## Milestones & Progress
+
+Our milestones are continually updated and detailed in the [IDM Workplan Document](https://docs.google.com/spreadsheets/d/1Venqgkcao2Lcje0mkCxr9u0H037aC3H5IxMfsoaeMoE).
+
+On the first monday of every sprint we will have a remote call, at 4:00 PM GMT, to report the project's current progress.
+If you care to join us you can do so by following this [link]().
+
+## Contributing
+
+All work is organised on GitHub throught this repository. There is also a [Project Board](https://github.com/ipfs-shipyard/pm-idm/projects/1) that we use for organisation, prioritisation and sprint planning.
+
+To manage our project, we adopted the Scrum methodology with two-week sprints. 
+Every sprint begins on a monday with the sprint planning meeting. Additionally, on the second monday of each sprint, there is a backlog grooming session.
+
+The best way to contribute would be to open a [GitHub Issue](https://github.com/ipfs-shipyard/pm-idm/issues) and, if you are willing to, open a [Pull Request](https://github.com/ipfs-shipyard/pm-idm/pulls) while targeting the respective issue. _Also please take in consideration that every commit should be following [Conventional Commits](https://conventionalcommits.org/) guidelines._
+
+You are also welcome to join us in a call at the beginning of every sprint by following this [link](). 
```

Comment:
Adapt this lane based on my comment above.

Reply:
If this line is related to line number 26, I think it's not necessary.